examples/activate-mongodb-atlas-resources.template.yaml

```diff
@@ -0,0 +1,341 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: MongoDB Resource activation - This template can be used to activate all MongoDB Atlas resources in current AWS account in the selected region. Note - This template should be used to activate only required resources by uncommenting 'TypeActivation' for the specific resource types, otherwise the stack may result in `Status Code 400 - Rate exceeded` error.
+Metadata:
+  cfn-lint: { config: { ignore_checks: [ W9002, W9003, W9006, E3001, E1010 ] } }
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: AWS Region
+        Parameters:
+          - Region
+Parameters:
+  Region:
+    Default: us-east-1
+    Description: The AWS Region where resources would be activated.
+    Type: String
+    AllowedValues:
+      - "us-east-1"
+      - "us-east-2"
+      - "ca-central-1"
+      - "us-west-1"
+      - "us-west-2"
+      - "sa-east-1"
+      - "ap-south-1"
+      - "ap-east-1"
+      - "ap-southeast-1"
+      - "ap-southeast-2"
+      - "ap-southeast-3"
+      - "ap-northeast-1"
+      - "ap-northeast-2"
+      - "ap-northeast-3"
+      - "eu-central-1"
+      - "eu-west-1"
+      - "eu-north-1"
+      - "eu-west-1"
+      - "eu-west-2"
+      - "eu-west-3"
+      - "eu-south-1"
+      - "me-south-1"
+      - "af-south-1"
+Resources:
+#  ActivateAlertConfigurationType:
+#    Type: AWS::CloudFormation::TypeActivation
+#    Properties:
+#      PublicTypeArn: !Join [ "", [ 'arn:aws:cloudformation:',!Ref "Region",'::type/resource/bb989456c78c398a858fef18f2ca1bfc1fbba082/MongoDB-Atlas-AlertConfiguration' ] ]
+#      Type: RESOURCE
+#      TypeName: MongoDB::Atlas::AlertConfiguration
+#      ExecutionRoleArn: !GetAtt MongoDBCustomResourceExecutionRole.Arn
+#  ActivateAuditingType:
+#    Type: AWS::CloudFormation::TypeActivation
+#    Properties:
+#      PublicTypeArn: !Join [ "", [ 'arn:aws:cloudformation:',!Ref "Region",'::type/resource/bb989456c78c398a858fef18f2ca1bfc1fbba082/MongoDB-Atlas-Auditing' ] ]
```

Comment:
I'm not sure I understand what the user experience would be like in that case?

Reply:
> Also, I think if the dependsOn works, we'll have to activate everything no? Since we have defined a dependency among all activations so now the user won't be able to pick and choose.

This proposal tries to handle the re-activate issue that you mentioned in the README. The idea is to allow users to select which resource they want to activate by providing input parameters to the template. 
> If the stack is created to activate already activated extensions in the region, you will run into failures as AWS does not allow to use `TypeActivation` to re-activate extensions.

Reply:
Discussed this offline. User can now activate all resources because of adding `dependsOn` among all resources so that activation runs synchronously. Hence, due to the dependency user cannot select specific resources to activate.